src/Repositories/FamilyRepository.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Choredo\Repositories;
+
+use Choredo\Filter;
+use Choredo\Pagination;
+use Choredo\Sort;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\Tools\Pagination\Paginator;
+
+/**
+ * Class FamilyRepository
+ * @package Choredo\Repositories
+ */
+class FamilyRepository extends EntityRepository
+{
+    /**
+     * @param Pagination $pagination
+     * @param Sort[] $sorts
+     * @param Filter[] $filters
+     * @return Paginator
+     */
+    public function getAll(Pagination $pagination, array $sorts = [], array $filters = []): Paginator
+    {
```

Comment:
Is it worth validating types on `$sorts` and `$filters`, you're calling mthods on them with no guarantee that they're being used correctly.